### ISSUE TITLE
Shading images using \tikz

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#275]
Originally reported by: **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**

----------------------------------------

As reported in http://tex.stackexchange.com/questions/147186/shading-images-in-beamer, the example 
````
\documentclass{beamer}
\usepackage{ifthen}
\usepackage{tikz}

\makeatletter
\newcommand{\includecoveredgraphics}[2][]{
    \ifthenelse{\the\beamer@coveringdepth=1}{
        \tikz
            \node[inner sep=0pt,outer sep=0pt,opacity=0.15]
                {\includegraphics[#1]{#2}};
    }{
        \tikz
            \node[inner sep=0pt,outer sep=0pt]
                {\includegraphics[#1]{#2}};%
    }
}

\makeatother

\begin{document}
\begin{frame}
\setbeamercovered{transparent}
\begin{itemize}[<+->]
\item First point.
\item Second point.
\item Third point.\includecoveredgraphics[width=2cm]{example-image-a}
\item Fourth Point \includecoveredgraphics[width=5cm]{example-image-b}
\end{itemize}
\end{frame}
\end{document} 
````
fails with v3.32 due to https://bitbucket.org/rivanvx/beamer/commits/39bc573c5bfc03fbd23ea3c16c420abfb6cf3d9b. Fix coming up.

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/275

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

End covering in tikzpicture environment but not raw \tikz (fixes #275)

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

Reverse allowing \pause in TikZ pictures

This reopens #275 but fixes #278 which is more important (introduce
no new issues).
